docs/acceptance_tests.md

```diff
@@ -0,0 +1,64 @@
+# Acceptance Tests
+
+## Basic Run
+
+Invoke `make test-acceptance` to running the epinio acceptance tests
+with its standard configuration.
+
+Invoke `make showfocus` to see if tests have been focused on.
+
+This target is automatically run as part of `make test-acceptance` as
+well.
+
+## Configuration
+
+The tests can be configured by a mixture of environment variables,
+files, and, of course, by editing the test go files. The latter to
+focus a run on specific tests, as per ginkgo's documentation.
+
+### Environment variables
+
+  1. `GINKGO_NODES`: The number of ginkgo nodes to distribute the
+     tests across. The default is `2`. The CI flows use `8`.
+
+  1. `FLAKE_ATTEMPTS`: The number of tries to perform when a test
+     fails, to ensure that the failure is real, and not a flake,
+     i.e. caused by a transient environmental condition. The default
+     is `2`.
+
+  1. `EPINIO_SKIP_PATCH`: When present and not empty the test startup
+     will skip over patching the epinio server pod of the test
+     cluster. This should be used when the test cluster still has a an
```

Comment:
```suggestion
     cluster. This should be used when the test cluster still has an
```

Reply:
Actually having an epinio server running is a prerequisite. The "reason" to do it is to run tests faster when you don't need to patch. For example, when the only thing you changed is the tests themselves and not the actual epinio code.